spec/features/web_intake/new_single_filer_spec.rb

```diff
@@ -130,6 +134,7 @@
     click_on "No"
 
     # Income from working
+    expect(intake.reload.current_step).to eq("/en/questions/job-count")
```

Comment:
I don't feel like we need so many of these same kinds of assertions. One or a few seems good.

Reply:
I was using these tests to ensure that the code actually worked in most cases that branch off in different ways to ensure that the backfill works in all cases.